src/extensions/magic_mount.rs

```diff
@@ -0,0 +1,53 @@
+mod chunk;
+
+type Error = Box<dyn std::error::Error + Send + Sync + 'static>;
+
+impl crate::application::CoLink {
+    pub(crate) async fn _mm_create_entry(
+        &self,
+        key_name: &str,
+        payload: &[u8],
+    ) -> Result<String, Error> {
+        let split_key = key_name.split('$').collect::<Vec<&str>>();
+        let token = split_key[split_key.len() - 1];
+        let key_name = split_key[0];
+        if token.eq("chunk") {
+            return self._create_entry_chunk(key_name, payload).await;
+        }
+        Err("invalid storage option".into())
```

Comment:
should also prompt `macro_type` in the error for debugging purposes. In fact, I would expect it to grow into a `match` statement with a default branch returning this error later; but it's fine to keep it as is for now.